justfile

```diff
@@ -110,7 +116,7 @@ legacy-e2e-tests *FLAGS:
 [no-exit-message]
 clean:
     -find . -name "*.pyc" -exec rm -rf {} \;
-    -find . -name .__pycache__ -exec rm -rf {} \;
+    -find . -name __pycache__ -exec rm -rf {} \;
```

Comment:
This is a mistake that was probably introduced by me. There will never be any folders with a leading dot. this doesn't exist.